splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java

```diff
@@ -85,6 +89,62 @@ public void testSinFunction() throws Exception{
         Assert.assertTrue("Incorrect rows returned!",rows>0);
     }
 
+	/**
+	 * Tests the ROUND function which rounds the input value to the nearest whole LONG
+	 * <p>
+	 * If the input value is NULL, the result of this function is NULL. If the
+	 * input value is equal to a mathematical integer, the result of this
+	 * function is the same as the input number. If the input value is zero (0),
+	 * the result of this function is zero.
+	 * <p>
+	 * The returned value is the closest (closest to positive infinity) long
+	 * value to the input value. The
+	 * returned value is equal to a mathematical long. The data type of the
+	 *
+	 * @throws Exception
+	 */
+	@Test
+	public void testRound() throws Exception {
+		ResultSet rs;
+		//testing round to Long
+		for(double val : roundVals){
+			rs = methodWatcher.executeQuery("values ROUND("+val+")");
+			Assert.assertTrue(rs.next());
+			Assert.assertEquals(Math.round(val),rs.getDouble(1),0.0);
```

Comment:
shouldn't we use 
Assert.assertEquals(Math.round(val),rs.getLong(1));
instead here, as both Math.round() and ROUND() with 1 parameter returns bigint?